pkg/mountutil/volumestore/volumestore.go

```diff
@@ -176,40 +241,62 @@ func (vs *volumeStore) Create(name string, labels []string) (*native.Volume, err
 
 // Get retrieves a native volume from the store
 // Besides a possible locking error, it might return ErrInvalidArgument, ErrNotFound, or a filesystem error
-func (vs *volumeStore) Get(name string, size bool) (*native.Volume, error) {
-	if err := identifiers.Validate(name); err != nil {
-		return nil, fmt.Errorf("malformed volume name %q: %w", name, err)
+func (vs *volumeStore) Get(nameOrDigest string, size bool) (*native.Volume, error) {
```

Comment:
Given how `List` is working, `Get` will now be able to retrieve from either name or digest.